doc/howtos/building-images.rst

```diff
@@ -195,7 +204,7 @@ compressed formats:
 To customize the image format, modify ``local.conf``, adding the variable
 ``OSTRO_VM_IMAGE_TYPES``, set to any combination of the following::
 
-    dsk dsk.xz dsk.vdi dsk.vdi.xz
+    dsk dsk.xz dsk.zip dsk.vdi dsk.vdi.xz dsk.vdi.zip
```

Comment:
Final thought on this one, we should describe how people can create images that can be flashed with bmaptool (and I mean here that a .bmap file would be produced along with the image). @kad what is the right approach? Would putting something like OSTRO_VM_IMAGE_TYPES = "dsk.xz.bmap" do the trick?